src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -1111,25 +1078,6 @@ def test_azconfig_feature(self, resource_group, location):
                                    self.check('[0].state', default_state)]).get_output_in_json()
         assert len(deleted) == 1
 
-        # Delete by pattern - this should delete 2 features Beta (label v1) and SuffixBeta
-        self.kwargs.update({
-            'feature': suffix_feature_pattern,
-            'label': any_label_pattern
-        })
-
-        deleted = self.cmd('appconfig feature delete --connection-string {connection_string}  --feature {feature} --label {label} -y',
-                           checks=[self.check('[0].locked', default_locked),
-                                   self.check('[0].key', entry_feature),
-                                   self.check('[0].description', updated_entry_description),
-                                   self.check('[0].label', entry_label),
-                                   self.check('[0].state', default_state),
-                                   self.check('[1].locked', default_locked),
-                                   self.check('[1].key', suffix_feature),
-                                   self.check('[1].description', default_description),
-                                   self.check('[1].label', null_label),
-                                   self.check('[1].state', default_state)]).get_output_in_json()
-        assert len(deleted) == 2
-
```

Comment:
Why you delete these testing code

Reply:
Due to a backend service behavior changes, those test cases were invalid now. The scenario of these testing code won't be supported any more.